### ISSUE TITLE
Getting ALS training loss with show_progress=False

### ISSUE THREAD

**Issue by V-H-R-Oliveira** [benfred/implicit#453]
Hi, i searched in the ALS source code some way of getting the training loss with the parameter `show_progress=False` enable, and I found nothing.

I don't know if it is possible to get this information without changing the source code or without call `calculate_loss` myself. I would appreciate any help.

Regards.

**Comment by benfred** [comment]
This isn't possible right now - we do have a `fit_callback` mechanism to run custom code in the training loop after each iteration, but its not passed the training loss metrics (so you'd have to call calculate_loss yourself).
